### DISCUSSION TITLE
Adding tooltip with additional values to bar chart

### DISCUSSION THREAD

**Discussion by LenaRampula** [h2oai/wave#654] [Q&A]
Is it possible to add additional values to the tooltip?
I'm plotting shap contributions of original features and would like to add the original feature value to the tooltip.

<img width="1306" alt="Screenshot 2021-03-21 at 11 58 15" src="https://user-images.githubusercontent.com/54841062/111902234-ba1bff80-8a3c-11eb-9ea9-d551f750c8e2.png">

**Comment by mtanco** [comment]
Related: https://github.com/h2oai/wave/issues/644
